doctr/models/recognition/sar/tensorflow.py

```diff
@@ -250,16 +284,17 @@ def call(
         target: Optional[List[str]] = None,
         return_model_output: bool = False,
         return_preds: bool = False,
-        **kwargs: Any,
```

Comment:
that always TF to pass "training=True/False" down to each module
Is there a reason why we should remove this?

Reply:
I don't think there's any reason to hold it at this time as you shouldn't get into this opportunity. Also, I'm not sure if it could cause problems when exporting ONNX. And the current implementation has an enormous potential for bugs through the massive use of kwargs down into the backbone. wdyt ? :)

Reply:
I said this because in TF, there is no `training` attribute. You pass `training=True` down to the call method, which is then passed to all submodules. So for instance here, if in one of the submodules, there is a batch norm layer, we're gonna be in big trouble 😅  (because those type of layers behave quite differently between training & test times)

Reply:
ok that's an really good argument :sweat_smile: